### PR TITLE
Implement Map Tile Generation and Serve Tiles via Mapbox Requests

### PR DESCRIPTION
**Pull request by sebmid**
#### Map Tile Generation and Serving:
- Added functionality to generate map tiles dynamically based on requests from Mapbox. This allows us to serve custom tiles to the Mapbox map component.

#### Dependency Adjustments:
- Introduced `rio_tiler` as a new dependency to facilitate efficient tile generation. This library provides a robust set of tools for working with raster data and will be integral to our tile-serving backend.

### Testing:
- Tested the tile generation endpoint with a variety of map zoom levels and coordinates to ensure reliability and performance. (Needs more testing)
- Ensured that the `OverlayView` component updates correctly with varying project IDs and coordinates.

### CODE REVIEW

File: backend/img2mapAPI/routers/georefProject.py
Comment (Trivinyx) [inline backend/img2mapAPI/routers/georefProject.py:1]: Move the logic code to utils.core.georef

**Review by Trivinyx — CHANGES_REQUESTED** [review]
All good, just move code to sparate the responibility of the code

**Review by Lennern — APPROVED** [review]
Meget kult, Sebastian! 
megetkult

File: backend/img2mapAPI/routers/georefProject.py
Comment (sebmid) [reply backend/img2mapAPI/routers/georefProject.py:1]: Done

**Pull request merged**